ui/components/MesheryPlayComponent.js

```diff
@@ -304,5 +277,5 @@ const mapStateToProps = (state) => {
 };
 
 export default withStyles(styles)(
```

Comment:
We can also remote withStyle and use styled component // @sudhanshutech

Reply:
Yeah that is being done by @Deathnote45 who is also assigned with me.
She's doing The MUI migration in a separate PR.

Reply:
Ohh I see, ok ok

Reply:
is @Deathnote45 is don for that other pr?